packages/components/src/styles/chip/chip.css

```diff
@@ -1,5 +1,5 @@
 .sd-chip {
-  @apply h-6 bg-primary-200 rounded-default text-black text-sm inline-flex items-center px-2 overflow-hidden;
+  @apply h-6 bg-primary-200 rounded-default text-black text-sm inline-flex items-center px-2 whitespace-nowrap overflow-hidden;
```

Comment:
Should this really be part of this PR? This looks like a fix for the sd-chip from Semantic Versioning perspective.

Reply:
The problem with wrapping was not visible in the chip history because there were no examples. Since the examples contain sd-chips and sd-flags and are only introduced here.
I can put this changes to a separate ticket and PR.